server/index.js

```diff
@@ -16,7 +18,9 @@ app.get('/photos/:id', (req, res) => {
   });
 });
 
-module.exports = app;
+app.listen(PORT, () =>console.log('You are listening to port: ', { PORT }));
```

Comment:
```suggestion
app.listen(PORT, () =>console.log('You are listening to port: ', PORT));
```

If what you have works, thats a-ok.... but I don't think I follow.